ui/app/src/components/Icons/GraphQL.tsx

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright (C) 2007-2020 Crafter Software Corporation. All Rights Reserved.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 3 as published by
+ * the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import React from 'react';
+import { createSvgIcon } from '@material-ui/core/utils';
+
+export default createSvgIcon(
+  <>
+    <path d="M3.99201 18.3828L3.18437 17.9165L12.1815 2.3334L12.9892 2.79969L3.99201 18.3828Z" />
+    <path d="M2.99942 16.6714H20.9937V17.6039H2.99942V16.6714Z" />
+    <path d="M12.3561 22.3918L3.35559 17.1952L3.82188 16.3876L12.8224 21.5842L12.3561 22.3918ZM20.1748 8.8481L11.1743 3.65149L11.6406 2.84385L20.6411 8.04046L20.1748 8.8481Z" />
+    <path d="M3.8244 8.84449L3.35811 8.03691L12.3586 2.8403L12.8249 3.64794L3.8244 8.84449Z" />
+    <path d="M20.0095 18.3826L11.0123 2.79949L11.82 2.3332L20.8171 17.9163L20.0095 18.3826ZM3.7136 7.4184H4.64618V17.8116H3.7136V7.4184ZM19.354 7.4184H20.2866V17.8116H19.354V7.4184Z" />
+    <path d="M12.1975 21.9972L11.7902 21.2917L19.6183 16.772L20.0256 17.4775L12.1975 21.9972Z" />
+    <path d="M21.5222 18.1152C20.9829 19.0534 19.7807 19.3736 18.8425 18.8343C17.9043 18.2949 17.5841 17.0927 18.1234 16.1545C18.6627 15.2163 19.8649 14.8961 20.8031 15.4354C21.747 15.9803 22.0672 17.177 21.5222 18.1152ZM5.87062 9.07588C5.3313 10.0141 4.12906 10.3343 3.19086 9.79497C2.25266 9.25565 1.93244 8.05341 2.47176 7.11521C3.01108 6.17701 4.21333 5.85679 5.15152 6.39611C6.08972 6.94105 6.40995 8.13768 5.87062 9.07588ZM2.47738 18.1152C1.93806 17.177 2.25828 15.9803 3.19648 15.4354C4.13468 14.8961 5.3313 15.2163 5.87624 16.1545C6.41556 17.0927 6.09534 18.2893 5.15714 18.8343C4.21333 19.3736 3.0167 19.0534 2.47738 18.1152ZM18.129 9.07588C17.5897 8.13768 17.9099 6.94105 18.8481 6.39611C19.7863 5.85679 20.9829 6.17701 21.5279 7.11521C22.0672 8.05341 21.747 9.25003 20.8088 9.79497C19.8706 10.3343 18.6683 10.0141 18.129 9.07588ZM11.9998 23.6151C10.9155 23.6151 10.0391 22.7387 10.0391 21.6545C10.0391 20.5702 10.9155 19.6938 11.9998 19.6938C13.0841 19.6938 13.9605 20.5702 13.9605 21.6545C13.9605 22.7331 13.0841 23.6151 11.9998 23.6151ZM11.9998 5.53657C10.9155 5.53657 10.0391 4.66017 10.0391 3.5759C10.0391 2.49164 10.9155 1.61523 11.9998 1.61523C13.0841 1.61523 13.9605 2.49164 13.9605 3.5759C13.9605 4.66017 13.0841 5.53657 11.9998 5.53657Z" />
```

Comment:
Can this line be broken up?

Reply:
I'm not sure. Not worth the effort anyway as these are sources generated by the design tool.